documentation/modules/ROOT/pages/tutorial.adoc

```diff
@@ -23,6 +23,12 @@ Debezium is a distributed platform that turns your existing databases into event
 
 Debezium {debezium-version} includes support for monitoring MySQL database servers with its xref:connectors/mysql.adoc[MySQL connector], and this is what we'll use in this demonstration. Support for other DBMSes will be added in future releases.
 
+== Running Debezium Sample Application
```

Comment:
There's already a "Note" box a few lines above which refers to the example in the _debezium-examples_ repo. So this one here would be a bit redundant. That said, this note does link towards the end of this document, and only from there it links to the repo. Perhaps the repo link should be added at the beginning, too?

Reply:
I do not care if this PR will be merged. I haven't seen this note yet (as I stuck on some manual steps) then I discovered tutorial that got everything running end to end in just 5 minutes. As comparing with 3 hours of poking with tutorial it was pure blessing. Any change that will give developers ability to skip pain of manually executing all commands will be awesome. 

Additionally I found that tutorial also requires people to understand kafka and how to connect to it. I found graphql tutorial more appropriate as it has end to end use case. 
Having end to end use case in form of java app with simple html page will help developers to get started.

Reply:
Ah I see it now:

```
If you have already completed the tutorial and you would like to go again through the setup quickly,
then you can use a https://docs.docker.com/compose/[Docker Compose] version of this tutorial located in our https://github.com/debezium/debezium-examples/tree/master/tutorial[examples repository].
```

This should be moved to the top IMHO. It will be cool to get some form of getting started experience directly on the website like in https://quarkus.io/get-started/

Reply:
@wtrocki I do agree there are areas of our documentation that could use some better flow.  Feel free to mock up something if time permits and send it our way to review; we'd love to take a look.